### ISSUE TITLE
[Programmatic Access – Make Code - Home page] Ensures every id attribute value is unique (image)

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3511]
**User Experience**:
The ID attribute uniquely identifies elements on a page. It does not make sense to duplicate an ID. Duplicate ID's can break the accessibility of labels for forms, table header cells, etc., by the second instance being skipped by screen readers, or by client-side scripts. They are common markup validation errors that can eliminate possible source of accessibility problems, when not breaking accessibility.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Run accessibility insights tool and select fast pass.
5. Verify the issue.

**Actual Result:**
ID attribute value is not unique on ‘Home page’.

**Expected result:**
ID attribute value should be uniquely defined under ‘Home page’

**WCAG Reference:**
[WCAG 4.1.1 - Parsing](https://www.w3.org/WAI/WCAG21/Understanding/parsing.html})

**Issue**: Ensures every id attribute value is unique (duplicate-id - https://dequeuniversity.com/rules/axe/4.0/duplicate-id?application=msftAI)

**Target application:** Microsoft MakeCode for micro:bit - https://makecode.microbit.org/#

**Element path**: defs:nth-child(7) > image

**Snippet:** <image id="blocklyDropdownArrowSvg" height="12px" width="12px" xlink:href="data:image/svg+xml;base64,PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIxMi43MSIgaGVpZ2h0PSI4Ljc5IiB2aWV3Qm94PSIwIDAgMTIuNzEgOC43OSI+PHRpdGxlPmRyb3Bkb3duLWFycm93PC90aXRsZT48ZyBvcGFjaXR5PSIwLjEiPjxwYXRoIGQ9Ik0xMi43MSwyLjQ0QTIuNDEsMi40MSwwLDAsMSwxMiw0LjE2TDguMDgsOC4wOGEyLjQ1LDIuNDUsMCwwLDEtMy40NSwwTDAuNzIsNC4xNkEyLjQyLDIuNDIsMCwwLDEsMCwyLjQ0LDIuNDgsMi40OCwwLDAsMSwuNzEuNzFDMSwwLjQ3LDEuNDMsMCw2LjM2LDBTMTEuNzUsMC40NiwxMiwuNzFBMi40NCwyLjQ0LDAsMCwxLDEyLjcxLDIuNDRaIiBmaWxsPSIjMjMxZjIwIi8+PC9nPjxwYXRoIGQ9Ik02LjM2LDcuNzlhMS40MywxLjQzLDAsMCwxLTEtLjQyTDEuNDIsMy40NWExLjQ0LDEuNDQsMCwwLDEsMC0yYzAuNTYtLjU2LDkuMzEtMC41Niw5Ljg3LDBhMS40NCwxLjQ0LDAsMCwxLDAsMkw3LjM3LDcuMzdBMS40MywxLjQzLDAsMCwxLDYuMzYsNy43OVoiIGZpbGw9IiNmZmYiLz48L3N2Zz4=">

**How to fix:**
Fix any of the following:
  Document has multiple static elements with the same id attribute: blocklyDropdownArrowSvg

Environment: Microsoft Edge version 87.0.658.0

====

This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#A11yMAS, #Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG4.1.1, #Benchmark, #Win10-Edge(Chromium), #A11yAuto

![WCAG4 1 1_Ensures every id attribute value is unique](https://user-images.githubusercontent.com/72871089/97537321-a2d46b00-19e4-11eb-9bb9-a8d9072e7d4e.jpg)
, #K4W

**Comment by abchatra** [comment]
Fixed
